.kitchen.yml

```diff
@@ -17,16 +17,27 @@ platforms:
 suites:
 - name: delete_from_list
   run_list:
+    - recipe[test::resource_status]
```

Comment:
The recipe adds a handler that reports the update status of each resource.  The tests check to see if a file was reported updated correctly by looking at the status in the report file.  Replaces the rspec idempotence checks.